contracts/utils/Bootstrap.sol

```diff
@@ -9,95 +9,111 @@ pragma solidity ^0.8.24;
 // /_/ |_/\___/_/|_\__,_/____/
 //
 // ──────────────────────────────────────────────────────────────────────────────
-// Nexus: A suite of contracts for Modular Smart Account compliant with ERC-7579 and ERC-4337, developed by Biconomy.
+// Nexus: A suite of contracts for Modular Smart Accounts compliant with ERC-7579 and ERC-4337, developed by Biconomy.
 // Learn more at https://biconomy.io. For security issues, contact: security@biconomy.io
 
 import { ModuleManager } from "../base/ModuleManager.sol";
 import { IModule } from "../interfaces/modules/IModule.sol";
 
+/// @title Bootstrap Configuration for Nexus
```

Comment:
Is this Bootstrap concept inspired by Rhinestone's ? Should credit them?

Reply:
I really don't know should ask Chirag when he's back he implemented this part

Reply:
I'll make a note of it and we can discuss it when Chirag returns.